runtime/src/main/scala/tailcall/runtime/model/Config.scala

```diff
@@ -318,7 +318,11 @@ object Config {
       RootSchema(query = other.query.orElse(query), mutation = other.mutation.orElse(mutation))
   }
 
-  final case class Type(doc: Option[String] = None, fields: Map[String, Field] = Map.empty) {
+  final case class Type(
+    doc: Option[String] = None,
+    `extends`: Option[ExtendsType] = None,
```

Comment:
```suggestion
    @jsonField("extends") extendsFrom: Option[ExtendsType] = None,
```

Reply:
Current implementation will require extends directive to be used as the following in Json format
```json
{
  extends: {
    extends: {
      type: "User"
    }
  }
}
```